### PR TITLE
User can see search results

### PR DESCRIPTION
**Pull request by tyladevon**
**Functionality:**
Complete User can enter search params (for climbing) and view the climbing route results. 

**Testing:**
Testing for feature is complete. No test for Poros.

**Related Issues:**
#14 and #11

### CODE REVIEW

File: app/controllers/trip_controller.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+class TripController < ApplicationController 
+  def new
+    @location = session[:destination]
```
Comment (madelynrr) [inline app/controllers/trip_controller.rb]: Necessary instance variable? Being used later?

File: spec/features/users/can_view_search_results_spec.rb [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+      choose 'Climbing'
+
+      click_on "Continue"
+
+      expect(current_path).to eq('/search/climbs/new')
+
+      select "5.7", from: :min_diff
+      select "5.12", from: :max_diff
+      select "Sport", from: :type
```
Comment (madelynrr) [inline spec/features/users/can_view_search_results_spec.rb]: Will need to change to checkbox.

**Comment by tyladevon** [comment]
The @location instance variable so to give access to the location in the session and is consistent with previous page's instance variable for location. Alternative to this would be to render a local inside the controller, but would recommend doing this for both views for consistency between views/controllers!

File: app/poros/climb_options.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
+class ClimbOptions
+  attr_reader :name, :climb_type, :rating, :stars, :pitches, :location, :url
+  def initialize(info)
+    @name = info[:name]
+    @climb_type = info[:type]
```
Comment (mintona) [inline app/poros/climb_options.rb]: Will the test be in the next PR?

File: app/poros/escape_facade.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    @params = params
+    @location = location
+  end 
+
+  def climb_options
+    results = EscapeService.new.get_climb_results(@params, @location)
+    results.map do |result|
+      if result[:type] == @params[:type] || result[:type] == "Sport, TR"
+        ClimbOptions.new(result)
+      end
```
Comment (mintona) [inline app/poros/escape_facade.rb]: Nice!

File: spec/features/search/user_can_fill_in_activity_preferences_spec.rb [hunk 5]
```diff
@@ -26,7 +26,7 @@
 
     click_on "Find Climbs!"
 
-    expect(current_path).to eq("/search/climbs")
-    expect(page).to have_css(".climb", count: 50)
+    expect(current_path).to eq('/trip/new')
+    expect(page).to have_css(".climbs")
```
Comment (mintona) [inline spec/features/search/user_can_fill_in_activity_preferences_spec.rb]: This test should go back to the actual number of climbs once the microservice is fixed.

**Pull request merged**